### PR TITLE
feat(brand-icons): add new warning brand icon

### PR DESCRIPTION
**Pull request by AlexandruHoisan**
add new warning brand icon

# Description

Add a new warning icon to the existing brand icons.

Storybook link: (once netlify has deployed link provide a link to the component)
Design link: [https://legalandgeneral.invisionapp.com/share/42ZNMDKXG3A#/436223089_AN_05-0_ChangeBankFailureNoRetry_Annuity_XL](https://legalandgeneral.invisionapp.com/share/42ZNMDKXG3A#/436223089_AN_05-0_ChangeBankFailureNoRetry_Annuity_XL)
Screenshot: 
Screenshot 2020-12-08 at 13 17 49
Screenshot 2020-12-08 at 13 17 36

# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [ ] I have provided an adequate amount of test coverage
- [ ] I have added the functionality to the [test app](./blob/master/docs/CONTRIBUTING.md#build-test-application)
- [ ] I have provided a story in storybook to document the changes
- [ ] I have provided documentation in the notes section of the story
- [ ] I have added any new public feature modules to public-api.ts
- [ ] I have [linked the new component](<(https://github.com/Legal-and-General/canopy/blob/master/docs/CONTRIBUTING.md#invision-dsm)>) to adobe DSM (if appropriate)

### CODE REVIEW

**Review by elenagarrone — APPROVED** [review]
Nice one!

File: projects/canopy/src/lib/icon/icons.interface.ts [hunk 1]
```diff
@@ -3,1039 +3,1039 @@
 
 export const lgIconAdd: Icon = {
   name: 'add',
-  data: `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32"><defs><path id="a" d="M17 15h8a1 1 0 010 2h-8v8a1 1 0 01-2 0v-8H7a1 1 0 010-2h8V7a1 1 0 012 0v8z"/></defs><use fill-rule="evenodd" xlink:href="#a"/></svg>`,
+  data: `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32"><defs><path id="a" d="M17 15h8a1 1 0 010 2h-8v8a1 1 0 01-2 0v-8H7a1 1 0 010-2h8V7a1 1 0 012 0v8z"/></defs><use fill-rule="evenodd" xlink:href="#a"/></svg>`
```
Comment (owensgit) [inline projects/canopy/src/lib/icon/icons.interface.ts:6]: Hmmm seems odd that the trailing comma has bene removed from all these lines. Prettier is supposed to add the comma's when they're missing on the pre-commit hook 🤔   I'll Slack you and see if there's something in your setup that's doing this

**Review by thatguynamedandy — DISMISSED** [review]
Looks good to me.

In case anyone was wondering the version of svg-to-ts changed and the files haven't been generated since, thats why it looks like both interface files have been re-written. There may be a slicker way of doing this centrally, but it would mean giving Github actions the ability to write files which needs some good security consideration.

File: projects/canopy/src/lib/icon/icons.interface.ts (hunk 1 above)
Comment (thatguynamedandy) [reply projects/canopy/src/lib/icon/icons.interface.ts:6]: I think this is due to the way the icons are setup. These interface files are actually generated on the pre-commit hook and so might not be setup to also work with prettier. Although I think they do have to be manually added to the commit so I wonder why it didn't run on them then. Could be you just need to run `npm install` locally as I think the trailing comma rule was added recently (cc @elenagarrone )

Comment (AlexandruHoisan) [reply projects/canopy/src/lib/icon/icons.interface.ts]: Had a look with Owen and fixed the prettier version on my machine.

**Review by owensgit — APPROVED** [review]

File: projects/canopy/src/lib/icon/icons.interface.ts (hunk 1 above)
Comment (owensgit) [reply projects/canopy/src/lib/icon/icons.interface.ts]: Alex was somehow using an older version of Prettier, despite doing an `npm install`. Updating it and running it again formatted the files correctly 👍

**Review by thatguynamedandy — APPROVED** [review]

**Pull request merged**